sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -77,7 +77,47 @@ final class DataFrameWriter private[sql](df: DataFrame) {
       case "ignore" => SaveMode.Ignore
       case "error" | "default" => SaveMode.ErrorIfExists
       case _ => throw new IllegalArgumentException(s"Unknown save mode: $saveMode. " +
-        "Accepted modes are 'overwrite', 'append', 'ignore', 'error'.")
+        "Accepted save modes are 'overwrite', 'append', 'ignore', 'error'.")
```

Comment:
We might consider aliasing `mode` as `saveMode` and deprecating `mode`.

/cc @rxin